src/context/mod.rs

```diff
@@ -79,6 +85,12 @@ impl Context for EmptyContext {
             identifier.to_string(),
         ))
     }
+    /// Builtin functions can't be disbaled for Empty Context.
+    fn disable_builtin_fn(&mut self) {}
+    /// Builtin functions are always disabled for Empty Context.
+    fn is_builtin_fn_disabled(&self) -> bool {
+        true
```

Comment:
It may be useful to have a separate new type `EmptyContextWithBuiltinFunctions` that behaves like the empty context but uses builtin functions by default, and builtin functions cannot be disabled.